doc/config_rsmeval.rst

```diff
@@ -132,6 +132,15 @@ RSMTool provides pre-defined sections for ``rsmeval`` (listed below) and, by def
 
     - ``evaluation by group``: Shows barplots with the main evaluation metrics by each of the subgroups specified in the configuration file.
 
+
+    - ``fairness_analyses``: Additional fairness analyses suggested in `Loukina, Madnani, & Zechner, 2019 <https://aclweb.org/anthology/papers/W/W19/W19-4401/>`_. The notebook shows:
+
+        - percentage of variance in squared error explained by subgroup membership
+        - percentage of variance in absolute error explained by subgroup membership
```

Comment:
So, we aren't using the signed difference like we said in the paper?

Reply:
API: done in the evaluation PR.

Reply:
Absolute: Oops. That's what it was supposed to be. I used absolute in a wrong sense. Will fix now.